source/admin_manual/clients.rst

```diff
@@ -0,0 +1,233 @@
+   oe-ns-eoh <pop3_client_workarounds>` workaround for POP3.
+
+Evolution
+---------
+
+-  Some versions don't support creating subfolders with mbox format.
+   Evolution in Ubuntu Gutsy, 2.12.0-0ubuntu5, does support creating
+   subfolders, at least when the parent folder is empty.
+
+Mulberry
+--------
+
+Seems to be OK.
+
+Claws-mail
+----------
+
+Everything works perfectly with Dovecot.
+
+Thunderbird
+-----------
+
+-  If you're using
+   :ref:`mbox <mbox_mbox_format>`,
+   :ref:`dbox <dbox_mbox_format>` or
+   :ref:`Maildir <maildir_mbox_format>` with ``:LAYOUT=fs`` ,
+
+   -  You should enable :dovecot_core:ref:`imap_client_workarounds =
+      tb-extra-mailbox-sep <imap_client_workarounds>` workaround for IMAP.
+      `Bug
+      report <https://bugzilla.mozilla.org/show_bug.cgi?id=29926>`__.
+
+-  If you're using :ref:`mbox <mbox_mbox_format>`
+
+   -  If you are not using a technique to allow folders that contain
+      both sub-folders and messages (e.g. see :ref:`mbox_child_folders`)
+      then you will have to disable "Server supports folders that
+      contain sub-folders and messages" setting from Thunderbird.
+      `Enhancement
+      request <https://bugzilla.mozilla.org/show_bug.cgi?id=284933>`__.
+
+-  Versions of Thunderbird from at least 17 (possibly earlier) up to
+   24.0 display incorrect new mail counts in the New Mail notification
+   box. This is due to a bug in Thunderbird's handling of the CONDSTORE
+   extension. See `Bug
+   Report <https://bugzilla.mozilla.org/show_bug.cgi?id=885220>`__ for
+   details and a client-side workaround.
+
+Mutt
+----
+
+-  New mutt versions supporting IDLE command will hang with Dovecot
+   versions earlier than v1.0beta3. Upgrade Dovecot or disable IDLE by
+   setting imap_idle=no in .muttrc.
+
+-  `Using mutt with IMAP <https://wiki.dovecot.org/Clients/mutt#>`__
+
+Pine
+----
+
+Seems to be OK.
+
+SquirrelMail
+------------
+
+-  Configuration asks IMAP server name for some workarounds. There has
+   been a Dovecot option since 1.4.6 and 1.5.1. For older SquirrelMail
+   versions, select the "other" option and remove the default
+   INBOX-prefix.
+
+Horde IMP
+---------
+
+Dovecot namespace detection works automatically with any recent version
+of IMP (4.1+).
+
+Quota support is now integrated into the 'imap' driver (as of
+horde-groupware V1.2), an example config of /imp/config/servers.php is:
+
+::
+
+   $servers['imap'] = array(
+       'name' => 'IMAP Server',
+       'server' => 'localhost',
+       'hordeauth' => false,
+       'protocol' => 'imap/notls',
+       'port' => 143,
+       'quota' => array('driver'=>'imap'),
+   );
+
+RoundCube Webmail
+-----------------
+
+Works fine.
+
+@Mail Webmail
+-------------
+
+Uses the namespace returned via Dovecot, full support via IMAP/POP3
+using `@Mail <http://atmail.com/>`__. Can also read mailbox quota via
+the getquotaroot IMAP command.
+
+RainLoop Webmail
+----------------
+
+Works fine.
+
+
+Negative UIDs
+*************
```

Comment:
What kind of client is this?

Reply:
The bad kind of client.

Reply:
OK. The actual question rather is: Why is this a subsection of RainLoop Webmail? Same with the "Fixing" sections below. https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#toc-entry-10 declares, that adornment styles enumerate the sections as encountered.

Something like (`@` just as an example, the guide recommends different characters):
```rst
Known Clients
@@@@@@@@@@@@@

Appsuite Webmail
----------------

...

Common Issues
@@@@@@@@@@@@@

Negative UIDs
-------------

...
```